packages/core-mobile/app/screens/drawer/notifications/Notifications.tsx

```diff
@@ -48,16 +54,23 @@ const Notifications = () => {
   }, [appState]) //switching to system settings and coming back must re-initiate settings check
 
   const renderNotificationToggles = useCallback(() => {
-    return notificationChannels.map(ch => {
```

Comment:
you should move the logic in here and check earn blocked, balance change blocked,... right now, the code still checks if earn is blocked, it will not render any notification toggles

Reply:
[Fix logic](https://github.com/ava-labs/core-mobile/pull/1841/commits/b5c7fcd3197e99a7178f96725258fdde6fd0e8a1)